src/amo/components/SearchForm.js

```diff
@@ -102,10 +158,12 @@ export function mapStateToProps(state) {
     addonType: validAddonTypes.includes(state.viewContext.context) ?
       state.viewContext.context : null,
     api: state.api,
+    suggestions: state.autocomplete.suggestions,
   };
 }
 
 export default compose(
+  withErrorHandler({ name: 'Autocomplete' }),
```

Comment:
Might not be the right name here. Should it be `Search` or `SearchFormBase` instead?